components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the multixact ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const MxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        The multixact ID is close to wraparound. This is about a multixact ID,
+        not about the normal transaction ID.
+      </p>
+      <p>
+        A multixact ID is an internal identifier used support row locking by
+        multiple transactions. Multixact IDs are created when transactions use{" "}
+        <code>SELECT ... FOR UPDATE</code> to lock and update tuples.
+      </p>
+      <p>
+        When the multixact ID wraparound happens, Postgres is unable to start a
+        new multixact and essentially makes the database down. Postgres usually
+        runs anti-wraparound autovacuums to prevent this from happening, but if
+        such VACUUMs are either not running or are running but unable to freeze
+        rows for some reasons, the transaction age (and the utilization) keeps
+        growing and eventually hit the point of wraparound. With Postgres 14+,
+        there is a special type of failsafe VACUUM which takes extraordinary
+        measures to avoid system-wide multixact ID wraparound failure. It is
+        recommended to take any actions to prevent the wraparound, or even
+        failsafe from happening.
```

Comment:
How about something like
```suggestion
        such VACUUMs are either not running, or are running but unable to freeze
        rows fast enough, the transaction age (and the utilization) keeps
        growing and eventually approaches the point of wraparound. With Postgres 14+,
        there is a special type of failsafe VACUUM which takes extraordinary
        measures to avoid system-wide multixact ID wraparound failure. Since
        a shutdown to prevent wraparound requires manual intervention, and
        a failsafe autovacuum ignores resource utilization constraints and can have
        significant performance impact, it is critical to ensure that freezing proceeds
        normally and mutlixact ID age is kept in check.
```
? I think this makes it a little more concrete.